spec/lib/areacodes_spec.rb

```diff
@@ -50,6 +50,30 @@
       end.sort_by {|r| -r.population }
       expect(cities).to eql(expected_records)
     end
+
+    it 'has only one state per area-code result' do
+      area_codes = described_class.details.map { |r| r.area_code }.uniq
+      area_codes.each do |ac|
+        major_cities = described_class.major_cities(ac)
+        states = major_cities.map{ |r| r.state_code }.uniq
+        expect(states.size).to eql(1), "Areacode: #{ac} has states: #{states}"
+      end
+    end
+
+    context 'fix bug - population missing for major cities for some area-codes' do
+      let(:affected_area_codes) do
+        [808, 838, 872, 934]
+      end
+
+      it 'has records with population' do
+        affected_area_codes.each do |ac|
+          major_cities = described_class.major_cities(ac)
+          expect(major_cities).to_not be_empty
+          expect(major_cities[0].population > 0).to be_truthy
```

Comment:
i _think_ this can be `expect(major_cities[0].population).to be > 0`, which i think will give a better error message when it fails. Otherwise it'll say expected false to be truthy or something like that